web_hide_custom_filter/models/ir_model.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class IrModel(models.Model):
+    _inherit = "ir.model"
+
+    show_custom_filter_groupby_group_ids = fields.Many2many(
```

Comment:
Let's keep it short.
```suggestion
    custom_search_group_ids = fields.Many2many(
```